src/playlist/playlistmanager.cpp

```diff
@@ -494,6 +494,11 @@ void PlaylistManager::RemoveItemsWithoutUndo(int id,
   playlists_[id].p->RemoveItemsWithoutUndo(indices);
 }
 
+void PlaylistManager::RemoveCurrentSong()
+{
```

Comment:
Sorry, I'm used to another format. And `clang-format` on the whole file does also change other code because not everything is formatted.
How should `make format` supposed to work?
>make: *** No rule to make target 'format'.  Stop.